app/models/prayer.rb

```diff
@@ -6,4 +6,5 @@ class Prayer < ActiveRecord::Base
   belongs_to :route, optional: true
 
   validates :recorded_at, presence: true
+  validates :resident, presence: true
```

Comment:
This gives us a better error when the resident_id is missing from a POST request to /prayers.

Reply:
Ok yeah I probably should have added a validation here